### PR TITLE
Preview UI changes

### PR DESCRIPTION
**Pull request by biyeun**
@benrudolph

* fixes scrollbar issues in tablet and phone mode (widths changing due to os / browser differences)
* some minor text formatting changes
* icon updates
screen shot 2016-11-28 at 4 23 54 pm

### CODE REVIEW

**Review by benrudolph — APPROVED** [review]
📱 👍

File: corehq/apps/style/static/app_manager/less/preview_app-main.less [hunk 1]
```diff
@@ -166,14 +166,16 @@ preview_app/less */
 }
 
 .btn-preview-refresh::after {
-  content: '✔';
-  color: green;
+  content: "\f00c";
```
Comment (benrudolph) [inline corehq/apps/style/static/app_manager/less/preview_app-main.less]: just curious, what's the benefit of doing this?

Comment (biyeun) [inline corehq/apps/style/static/app_manager/less/preview_app-main.less]: it uses the font awesome checkmark icon instead. keeps things standardized. the other checkmark changes depending on the system and just looks awkward against the other icons

**Comment by benrudolph** [comment]
nice, the one question thing for formplayer is just cause the setting persists in localstorage. thinking about it again, i think it makes sense to namespace the settings with the environment they're in (app preview vs. web apps). i can write a PR that does that

**Comment by benrudolph** [comment]
test fail real
```
======================================================================
FAIL: Tests that the diffs of the template files match what we have
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/commcare-hq-ro/corehq/apps/app_manager/tests/test_app_manager_v2_diffs.py", line 167, in test_partial_02_diffs_match
    self.partial_v2_dir
  File "/mnt/commcare-hq-ro/corehq/apps/app_manager/tests/test_app_manager_v2_diffs.py", line 115, in _test_diffs_util
    test_file, filename_v1, filename_v2
AssertionError: 
*************************************
```

**Comment by biyeun** [comment]
haaa...my own test

**Pull request closed without merging**